goblet/common_cloud_actions.py

```diff
@@ -11,17 +11,20 @@
     get_default_location,
     get_credentials,
 )
+from goblet.utils import get_python_runtime
 
 log = logging.getLogger("goblet.deployer")
 log.setLevel(logging.INFO)
 
 
-def create_cloudfunction(client, req_body, config=None):
+def create_cloudfunction(client: Client, params, config={}):
```

Comment:
config can stay none. when gconfig is iniatided and config is none it will just return what is in config.json

Reply:
Ok!